dcc-storage-core/pom.xml

```diff
@@ -53,13 +53,27 @@ ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF S
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
       <version>19.0</version><!--$NO-MVN-MAN-VER$-->
+    </dependency>   
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+      <version>2.3.4</version>

```

Comment:
build was failing without the version being specified - which was really weird. But since it's actually a dependency of jackson-binding, I was able to just remove it.